.gitmodules

```diff
@@ -41,3 +41,9 @@
 [submodule "meta-browser"]
 	path = meta-browser
 	url = git://github.com/OSSystems/meta-browser.git
+[submodule "renesas-rcar-gen3"]
+	path = renesas-rcar-gen3
+	url = https://github.com/slawr/renesas-rcar-gen3.git
```

Comment:
@zeenix Are you saying you will not accept the PR without your changes? I think the changes would need to be defined in much more detail but that question is the first gate I think.